types/geometric/index.d.ts

```diff
@@ -7,194 +7,218 @@ export as namespace geometric;
+ * 1]]. Polygons can be closed – the first and last vertex are the same – or open.
+ */
+export type Polygon = Point[];
 
 // -------------------------------POINTS-------------------------------------------//
 
-    /**
-     * Returns the coordinates resulting from rotating a point about an origin by an angle in degrees. If origin is
-     * not specified, the origin defaults to [0, 0].
-     */
-    export function pointRotate(point: Point, angle: number, origin?: Point): Point;
+/**
+ * Returns the coordinates resulting from rotating a point about an origin by an angle in degrees. If origin is
+ * not specified, the origin defaults to [0, 0].
+ */
+export function pointRotate(point: Point, angle: number, origin?: Point): Point;
 
-    /**
-     * Returns the coordinates resulting from translating a point by an angle in degrees and a distance.
-     */
-    export function pointTranslate(point: Point, angle: number, distance: number): Point;
+/**
+ * Returns the coordinates resulting from translating a point by an angle in degrees and a distance.
+ */
+export function pointTranslate(point: Point, angle: number, distance: number): Point;
 
 // -------------------------------LINES-------------------------------------------//
 
-    /**
-     *  Returns the angle of a line, in degrees, with respect to the horizontal axis.
-     */
-    export function lineAngle(line: Line): number;
+/**
+ *  Returns the angle of a line, in degrees, with respect to the horizontal axis.
+ */
+export function lineAngle(line: Line): number;
 
-    /**
-     * Returns an interpolator function given a line [a, b]. The returned interpolator function takes a single
-     * argument t, where t is a number ranging from 0 to 1; a value of 0 returns a, while a value of 1 returns b.
-     * Intermediate values interpolate from a to b along the line segment.
-     */
-    export function lineInterpolate(line: Line): LineInterpolator;
+/**
+ * Returns an interpolator function given a line [a, b]. The returned interpolator function takes a single
+ * argument t, where t is a number ranging from 0 to 1; a value of 0 returns a, while a value of 1 returns b.
+ * Intermediate values interpolate from a to b along the line segment.
+ */
+export function lineInterpolate(line: Line): LineInterpolator;
 
-    export type LineInterpolator = (t: number) => Point;
+export type LineInterpolator = (t: number) => Point;
 
-    /**
-     * Returns the length of a line.
-     */
-    export function lineLength(line: Line): number;
+/**
+ * Returns the length of a line.
+ */
+export function lineLength(line: Line): number;
 
-    /**
-     * Returns the midpoint of a line.
-     */
-    export function lineMidpoint(line: Line): Point;
+/**
+ * Returns the midpoint of a line.
+ */
+export function lineMidpoint(line: Line): Point;
+
+/**
+ * Returns the vertices resulting from rotating a line about an origin by an angle in degrees. If origin is not
+ * specified, the origin defaults to [0, 0].
```

Comment:
The default origin here should be the midpoint
> Returns the coordinates resulting from rotating a line about an origin by an angle in degrees. If origin is not specified, the origin defaults to the [midpoint](https://github.com/HarryStevens/geometric#lineMidpoint) of the line.

https://github.com/HarryStevens/geometric#lineRotate